buildSrc/build.gradle.kts

```diff
@@ -33,10 +33,3 @@ dependencies {
   implementation(libs.test.rerun.plugin)
   implementation(libs.nessie.runner.plugin)
 }
-
```

Comment:
No longer needed

Reply:
will it use the version from the JVM that runs gradle?

Reply:
yup